db/migrate/20180927095205_devise_create_customers.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class DeviseCreateCustomers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :customers do |t|
```

Comment:
When creating the customers table, we should ask for firstname and lastname, no?